GameData/RP-0/Contracts/Milestones/Orbital Recovery.cfg

```diff
@@ -110,3 +99,226 @@ CONTRACT_TYPE
 		}
 	}
 }
+
+CONTRACT_TYPE
+{
+	name = second_OrbitRecover
+	title = Reach Orbit & Return Safely to Earth (Moderate)
+	group = Milestones
+	agent = Federation Aeronautique Internationale
+
+	description = Reentry is brutally hot and dangerous but it can be survivable with the right technology and equipment. We want you to design, build, and launch a craft that can survive a reentry and make a soft landing or splashdown on Earth after achieving a typical orbit (roughly 200km). This is a step up in the testing to ensure that our equipment works under expected orbital mission parameters. Note that this contract eill be unavailable after completing the first crewed orbit.
+	genericDescription = Recover a craft from an typical crewed orbit descent
+	
+	synopsis = Recover a craft from an typical crewed orbit descent
+
+	completedMessage = Congratulations! The craft has survived reentry, and engineers are examining it very keenly indeed.
+
+	sortKey = 104
+
+	cancellable = false
+	declinable = false
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 1
+	maxSimultaneous = 1
+	deadline = 365 * RP1DeadlineMult()  // 1 year
+
+	targetBody = HomeWorld()
+
+	// ************ REWARDS ************
+	// Same as initial return
+	prestige = Significant   // 1.25x
+	advanceFunds = 50000 * @RP0:globalHardContractMultiplier
+	rewardScience = 2
+	rewardReputation = 10
+	rewardFunds = 50000 * @RP0:globalHardContractMultiplier
+	failureReputation = 10
+	failureFunds = @advanceFunds * 0.5
+
+	// ************ REQUIREMENTS ************
+	//Requires the first orbital return to be completed
+	REQUIREMENT
+	{
+		name = CompleteContract
+		type = CompleteContract
+		contractType = first_OrbitRecover
+	}
+
+	//If you do manned orbital missions first then saftey is not that important to you so no contract
+	//Allow crewed sub orbital contracts to be done as this is orbital velocity related
+	REQUIREMENT
+	{
+		name = CompleteOrbit
+		type = CompleteContract
+		contractType = first_OrbitCrewed
+		invertRequirement = true
+	}
+
+	PARAMETER
+	{
+		name = OrbitReturn
+		type = VesselParameterGroup
+		title = Recover a craft from Orbit
+
+		PARAMETER
+		{
+			name = NewVessel
+			type = NewVessel
+			title = Launch a New Vessel
+			hideChildren = true
+		}
+		PARAMETER
+		{
+			name = HasNoCrew
+			type = HasCrew
+			maxCrew = 0
+			minCrew = 0
+			title = Uncrewed
+			hideChildren = true
+		}
+		PARAMETER
+		{
+			name = Orbit
+			type = Orbit
+			minPeA = 200000
+			disableOnStateChange = true
+			title = Achieve Orbit within the parameters
+			
+			//Do we wat to kee this? I lean to yes but is not really necessary
```

Comment:
Might as well keep it since it will prevent you from doing a 180 and burning retrograde with the same stage. Not sure if increasing the duration to 10 minutes for the 3rd contract does anything useful though.

Reply:
I put in the 10 minute duration as a means to prevent achieving the high apogee and then deorbiting. 10 minutes will at least ensure that they achieve an actual altitude sufficient to do a more strenuous test. Perhaps this can be achieved by added an altitude parameter?

Reply:
All right, keep the 10 minute requirement then. I would appreciate it if you removed those 2 comment-questions though.

Reply:
Done